frontend/src/components/FacilityBooking/HourBlocks.tsx

```diff
@@ -2,6 +2,13 @@ import React from 'react'
 
 import { MainTimeContainer, TimeContainer } from './BlockStyles.styled'
 
+/**
+ * @returns A column of each hour from 0000 to 2400, to be displayed next to DailyContainer containing time blocks
```

Comment:
erm for me ithe description wasn't immediately obvious to me haha; perhaps could describe it as a column that labels the hour blocks of the `DailyContainer`?

Reply:
updated description